app/kumactl/cmd/install/testdata/install-control-plane.with-egress.golden.yaml

```diff
@@ -1641,6 +1641,7 @@ spec:
         - name: control-plane
           image: "docker.io/kumahq/kuma-cp:0.0.1"
           imagePullPolicy: IfNotPresent
+          hostNetwork: false
```

Comment:
looks like this is done at the container level when it's meant to be done at the pod level which is why the e2e tests fail

Reply:
Hi you are right, I changed it :)